### PR TITLE
add nu5036 error related to bad license uri in 4.9.0

### PR DESCRIPTION
**Pull request by rrelyea**
server will reject packages with badly escaped licenseuri (4.9.0 bug).
when somebody tries to submit such a package, they will block it...and point to nu5036 error page.
this page's goal is to help people find the right updated version of "nuget pack" if available...or find out how to undo their use of license expression/file temporarily.

@nkolev92 @agr 
@anangaur @karann-msft @skofman1

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-nu5036/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1163) |
[docs/reference/errors-and-warnings/NU5036.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-nu5036/docs/reference/errors-and-warnings/NU5036.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5036?branch=pr-en-us-1163) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811152140271505-1163%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fJ29a1FLX4%2B7RG%2FbhX2WHIHzDqoq3GKPpfQYw7VPf60%3D&st=2018-11-15T21%3A36%3A26Z&se=2018-12-16T21%3A41%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU5036.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ms.author: karann
+ms.date: 8/8/2018
+ms.topic: reference
+ms.reviewer: anangaur
+f1_keywords: 
+  - NU5036
+---
+
+# NuGet Error NU5036
+<pre>This package has an improperly escaped Url in LicenseUrl</pre>
```
Comment (nkolev92) [inline docs/reference/errors-and-warnings/NU5036.md]: This package has an improperly escaped LicenseUrl. 

The Url seems superfluous.

File: docs/reference/errors-and-warnings/NU5036.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### Issue
+
+LicenseUrl metadata value in the nuspec file should be properly escaped.
+Some versions of nuget pack functionality have a problem which will be updated asap.
```
Comment (nkolev92) [inline docs/reference/errors-and-warnings/NU5036.md]: Some versions of nuget pack functionality have a problem. 

The will be updated part is covered in the solution.

Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU5036.md]: We can consider adding the versions in which it is broken.

File: docs/reference/errors-and-warnings/NU5036.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### Solution
+
+Use a fixed version of nuget pack functionality:
+* "NuGet.exe pack" - fixed in 4.9.1
```
Comment (anangaur) [inline docs/reference/errors-and-warnings/NU5036.md]: 4.9.1 is still not available. So lets mention that is broken in 4.9.0 and fix will be available in 4.9.1

File: docs/reference/errors-and-warnings/NU5036.md (hunk 1 above)
Comment (anangaur) [inline docs/reference/errors-and-warnings/NU5036.md]: Suggestion: This package has improperly escaped LicenseUrl

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[d2d431b](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1163/commits/d2d431b2f675a866bd900c8348f57ac935c2122c)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[d2d431b](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1163/commits/d2d431b2f675a866bd900c8348f57ac935c2122c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-nu5036/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1163) |
[docs/reference/errors-and-warnings/NU5036.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-nu5036/docs/reference/errors-and-warnings/NU5036.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5036?branch=pr-en-us-1163) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201901152302065225-1163%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8SwdWWdHpz%2FwlGq0Fz3J20QRnRB7pK4MKfTuus8%2FOfA%3D&st=2019-01-15T22%3A58%3A10Z&se=2019-02-15T23%3A03%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[51a06d6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1163/commits/51a06d6b33baa5e7fda59b4dfd89405d403c0ac2)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[51a06d6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1163/commits/51a06d6b33baa5e7fda59b4dfd89405d403c0ac2)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[51a06d6](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1163/commits/51a06d6b33baa5e7fda59b4dfd89405d403c0ac2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-nu5036/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5036?branch=pr-en-us-1163) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-nu5036/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1163) |
[docs/reference/errors-and-warnings/NU5036.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-nu5036/docs/reference/errors-and-warnings/NU5036.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5036?branch=pr-en-us-1163) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201901152304421434-1163%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Ofpr6Qo109tmCfTKzhDVEvFkXSliS38k0ZEcKksEtMI%3D&st=2019-01-15T23%3A00%3A47Z&se=2019-02-15T23%3A05%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**